okhttp/src/main/java/okhttp3/HttpUrl.kt

```diff
@@ -828,6 +831,9 @@ class HttpUrl internal constructor(builder: Builder) {
       encodedPathSegments.add("") // The default path is '/' which needs a trailing space.
     }
 
+    /**
+     * @param scheme either "http" or "https"
```

Comment:
period at the end of this too. https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

Reply:
Period added